components/auth.mdx

```diff
@@ -0,0 +1,6 @@
+---
+title: "Authentication"
+description: "Secure your data with token-based authentication"
```

Comment:
what is this

Reply:
Was going to explain the queryHeaders more here but ended up deleting this page. We discuss queryHeaders a little bit in the selfhost docs.